flow/project.py

```diff
@@ -3744,24 +4015,41 @@ def operation_function(job):
         except KeyError:
             raise KeyError("Unknown operation '{}'.".format(args.operation))
 
-        for job in jobs:
-            operation_function(job)
+        for aggregate in self._get_aggregates(args.operation):
+            if self._is_selected_aggregate(aggregate, aggregates):
+                operation_function(aggregate)
 
     def _select_jobs_from_args(self, args):
-        "Select jobs with the given command line arguments ('-j/-f/--doc-filter')."
-        if args.job_id and (args.filter or args.doc_filter):
+        "Select jobs with the given command line arguments ('-j/-f/--doc-filter/--jobid')."
+        # Collecting command line argument variables
+        args_job_id = getattr(args, 'job_id', None)
+        args_jobid = getattr(args, 'jobid', None)
```

Comment:
I'm very confused, why are there two supported arguments here, `job_id` and `jobid`?

Reply:
`exec` uses `jobid` but every other command uses `job_id`.
We could, however, deprecate `jobid` and ensure that we're consistent.

Reply:
Oh gosh, I see that now. This was almost certainly a mistake. We should decide whether to deprecate `jobid` over the next 2 minor versions, or just break the API. This is a subtle enough change that I would favor breaking the API in the next version, but I'd like to get consensus on that. I believe it might be safe to just break this (and document it in the changelog) because `exec` is almost never directly called by users, and I would guess that most users would provide `-j` and not `--jobid`. The `exec` command is functionally like an internal API because most users only indirectly use it through `run`.

Reply:
@bdice I am in favor of just fixing this without a deprecation cycle.

Reply:
This is resolved, so I'm going to resolve the conversation.